packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dedicatedCloud.service.js

```diff
@@ -791,7 +791,7 @@ class DedicatedCloudService {
           serviceName,
           networkAccessId: entry.id,
         },
-        data: pick(entry, 'description'),
+        data: pick(entry, ['description', 'network']),
```

Comment:
```suggestion
        data: {
        description: entry.description, 
        network: entry.network 
       }
```
Maybe we can skip lodash use here

Reply:
@marie-j  yes we can skip but we are using loadash functions here for almost everything and for this also loadash pick method was there, that's why I didn't remove it.